sources2csr/derived_values.py

```diff
@@ -1,15 +1,16 @@
 from collections import defaultdict
-from operator import attrgetter
-from typing import List, Dict
+from typing import List, Dict, Set
 
 from dateutil.relativedelta import relativedelta
 
 from csr.csr import CentralSubjectRegistry, Diagnosis
+from sources2csr.ngs import NGS
 
 
-def add_derived_values(subject_registry: CentralSubjectRegistry) -> CentralSubjectRegistry:
+def add_derived_values(subject_registry: CentralSubjectRegistry, ngs_set: Set[NGS]) -> CentralSubjectRegistry:
```

Comment:
Maybe better to create a separate function `add_ngs_data` as these are not strictly derived from the subject registry data.